docs/tutorials/index.md

```diff
@@ -123,6 +123,22 @@ The [Druid router process](../design/router.md), which serves the [Druid console
 
 It takes a few seconds for all the Druid processes to fully start up. If you open the console immediately after starting the services, you may see some errors that you can safely ignore.
 
+## Alternative: Running with Docker
```

Comment:
I don't think this makes sense here and should have its own tutorial dedicated to docker

Reply:
I think this is a good idea to add it as a dedicated tutorial. Do you think is helpful to have it as a reference also in the `tutorials/index.md` that can reference more in depth documentation? I think that for a `Getting started / Quickstart`, a section with `Docker` is also helpful. 

My intention was to be able to start this conversation here and also extend the documentation as needed.

What do you think?